boost.sh

```diff
@@ -623,7 +625,7 @@ using clang : 5.0~arm
 <compileflags>-isystem <compileflags>$ANDROID_NDK_ROOT/sysroot/usr/include
 <compileflags>-isystem <compileflags>$ANDROID_NDK_ROOT/sysroot/usr/include/arm-linux-androideabi
 <compileflags>-DANDROID
-<compileflags>-D__ANDROID_API__=16
+<compileflags>-D__ANDROID_API__=19
```

Comment:
Why did you increase version here? I thought you want to build with minimum possible API?